dm/loader/lightning.go

```diff
@@ -216,16 +216,18 @@ func (l *LightningLoader) runLightning(ctx context.Context, cfg *lcfg.Config) er
 	var opts []lightning.Option
 	if l.cfg.MetricsFactory != nil {
 		// this branch means dataflow engine has set a Factory, the Factory itself
-		// will register and deregister metrics, so we must use NoopRegistry
-		// to avoid duplicated registration.
+		// will register and deregister metrics, but lightning will expect the
```

Comment:
Do you think we need to find a better way for registering and unregistering the metric?